packages/web-react/src/components/UNSTABLE_Toggle/index.ts

```diff
@@ -0,0 +1,4 @@
+export * from './UNSTABLE_Toggle';
+export { default as UNSTABLE_Toggle } from './UNSTABLE_Toggle';
+export * from './useToggleStyleProps';
+// some comment
```

Comment:
Please, remove this comment. I accidentally leave it here, when testing the problem with the installation.